services/GoogleSitemapService.cfc

```diff
@@ -65,12 +67,21 @@ component {
 				}
 			}
 
+			if ( page.sitemap_priority=="inherit" ) {
+				if ( !structKeyExists( inheritedPageSitemapPriority, page.parent_page ) ) {
+					page.sitemap_priority = _getSitemapPriorityForPage( page.id ).sitemap_priority;
```

Comment:
This needs amending to fit with the change to the method signature

Reply:
Hi Dom, 

Have updated the method call

Thank you
Nelson